config/routes.rb

```diff
@@ -2681,11 +2680,6 @@
         x_show
         zone_edit
         zone_field_changed
-        ldap_region_add
-        ldap_region_edit
-        ldap_region_form_field_changed
-        ldap_domain_edit
-        ldap_domain_form_field_changed
         ls_select
         ldap_entry_changed
```

Comment:
Shouldn't `ldap_entry_changed` go away as well?

Reply:
Since you're removing `ldap_entry_changed()` routine, the `ldap_entry_changed` route needs to go away as well.

Reply:
👍